content/overview/technology/responsibilities.md

```diff
@@ -15,15 +15,17 @@ Here's a chart to illustrate this in three example use cases:
 
 <img src="/img/boundaries.svg" alt="Diagram of responsibilities, described in text below" width="450" />
 
-App #1 uses a [Cloud Foundry standard buildpack]({{< relref "docs/getting-started/concepts.md#buildpacks" >}}). (A buildpack provides support for a programming language.) The customer is only responsible for the app code and its dependencies.
+<!-- Source for this diagram is https://docs.google.com/drawings/d/1UBiOteSPXpA72KE52Kh-j7aYr73zTkzJ_oMuw5F293I/edit -->
```

Comment:
Replace with
`{{/* Source for this diagram is https://docs.google.com/drawings/d/1UBiOteSPXpA72KE52Kh-j7aYr73zTkzJ_oMuw5F293I/edit */}}`

This way this link will not show up in the HTML source output.